Bradm/web summit (#3339)
FusionAuth/fusionauth-site: astro/src/content/blog/fusionauth-web-summit.mdx

We asked Mike Butcher, Techcrunch Editor-at-Large and Web Summit veteran for his tips on engaging with the media. So here they are:

1. Pinpoint your journalist before the event with an email/contact to set-up a meeting or offer them an story exclusive so the story comes out whilst your are there

2. Scour the agenda to see where a journalist is moderating a panel/interview and try and grab them as they leave the stage

3. “Goal hang” where the media enter the main venue from the VIP/Press area

4. Repeat steps 2 & 3

Make sure you follow Mike - @mikebutcher and check out [The Europas Awards](https://www.theeuropas.com/) and event he created which runs during Web Summit.

![Enter image alt description](/img/blogs/fusionauth-web-summit/mike-butcher-wide.jpg)

## **Making the Most of Social Events and Parties**

### **Side Events**

As happens with most events of such a size, there are a ton of non-official but amazing side events every night (and some early morning ones too). Some of these events are extremely niche but you just might find your tribe there.

Luma published a list of them [here](https://lu.ma/websummit) last year, but keep an eye on Meetup and Eventbrite too.

> Side events tend to fill up super quickly as people tend to register for everything, and choose what they attend closer to the time. Most experienced side event organizers know this, so if there’s a waitlist, join it as you’ll probably get in.

Reviewer: ```suggestion
Side events tend to fill up super quickly as people tend to register for everything, and choose what they attend closer to the time. Most experienced side event organizers know this, so if there’s a waitlist, join it. You’ll probably get in.
```

> The larger "official" parties will be overrun, try to go for the smaller, themed events that are in your area of interest. This will alow you to find people that are potential valuable connections.

Reviewer: ```suggestion
The larger "official" parties will be overrun so go for the smaller, themed events that are in your area of interest. This will allow you to find people that are potential valuable connections.
```

> ### **Night Summit**

Reviewer: Remove **

The Night Summit events put on as part of Web Summit are a heap of fun. Each night hosted in different regions of the city, they are a heaving buzz of energy and excitement . The opportunity to meet with entrepreneurs from all over the globe is amazing, but don’t expect to meet that perfect partner or investor here. There’s too many people and it’s too random.

> ![Enter image alt description](/img/blogs/fusionauth-web-summit/night-summit-wide.png)

Reviewer: ```suggestion
![A picture of the Night Summit.](/img/blogs/fusionauth-web-summit/night-summit-wide.png)
```

### **Tips for Social Events**

Social events are an excellent opportunity to network in a more relaxed setting.

> Wear your Web Summit badge all the time. I once connected with some people that saw I was from SendGrid whilst sitting by myself at a restaurant. I ended up joining their table and met and amazing group of people.

Reviewer: ```suggestion
Wear your Web Summit badge all the time. I once connected with some people that saw I was from SendGrid whilst sitting by myself at a restaurant. I ended up joining their table and met an amazing group of people.
```

**Don't Drink to Much**

> You won’t be at your best and it will make the next day very long, trust me ;)

Reviewer: ```suggestion
You won’t be at your best and it will make the next day very long, trust me ;).
```

**Don’t Just Sell**

> Ask people what they are working on. Trust me great results come from listening to people.

Reviewer: ```suggestion
Ask people what they are working on. Trust me. 

Great results come from listening to people.
```

**Stay Humble and Thankful**

E.g. “It was great meeting you, and good luck with your venture.”

While it’s tempting to attend every social event, ensure you’re not compromising your productivity. Strike a balance between work and leisure to get the most out of the experience.

> ## **Using Social Media Effectively**

Reviewer: Remove **

Share your experiences on social media using event hashtags and official Web Summit accounts. Engage with other attendees online (on socials and in the app) to expand your network. Tag people with accounts and hashtags or any people featured in your posts.

> ### **Official Web Summit Accounts to Tag**

Reviewer: Remove **

X.com - [https://x.com/WebSummit](https://x.com/WebSummit)

Instagram - [https://www.instagram.com/websummit/](https://www.instagram.com/websummit/)

LinkedIn - [https://www.linkedin.com/company/web-summit/](https://www.linkedin.com/company/web-summit/)

TikTok - [https://www.tiktok.com/@websummit](https://www.tiktok.com/@websummit)

YouTube - [https://www.youtube.com/@websummit](https://www.youtube.com/@websummit)

> ### **Hashtags**

Reviewer: Remove **

Hashtags to be used are as follows:

#websummit #websummit2024 #websummitlisbon

**Use Shorts**

Instagram Reels, TikTok, YouTube Shorts and LinkedIn Shorts (if they are available to you) are well worth posting. You’re a great chance to be shared by the official Web Summit account, and who knows what investor, partner or customer might see it.

***“Document. Don’t create”*** - Gary Vaynerchuck

Here’s a [great article from Gary Vee](https://garyvaynerchuk.com/creating-content-that-builds-your-personal-brand/) about approaching social media.

> Even if you don’t post regularly, you can post a reel after the event documenting your experience. Here’s [one](https://www.youtube.com/watch?v=k2UGhmxvcbU) I did a few years ago.

Reviewer: ```suggestion
Even if you don’t post regularly, you can post a reel after the event documenting your experience. [Here’s one](https://www.youtube.com/watch?v=k2UGhmxvcbU) I did a few years ago.
```

> ### **Video Interviews**

Reviewer: Remove **

> Come across a great customer or partner, take a few minutes to interview them. You will need a good mic as it’s noisy. Whilst your phone’s camera will be good enough, the mic will not and would leave the footage unusable. Something simple like this mic on Amazon would be fine.

Reviewer: ```suggestion
If you come across a great customer or partner, take a few minutes to interview them. You will need a good mic as it’s noisy. Whilst your phone’s camera will be good enough, the mic will not and would leave the footage unusable. Something simple like this mic on Amazon would be fine.
```

Reviewer: Looks like he forgot the link, I'd probably just remove the last sentence unless you have it lying around in a google doc.

> ### **Things to do in Lisbon**

Reviewer: Remove **

Lisbon is one of the great cities of Europe. Here’s my must do list:

**Belém** - by the seaside with the best Pastéis ever!! 

**Bairro Alto** - fun neighborhood cobblestone alleys of bars and restaurants

**Time Out Market** - amazing food court showcasing the best of Lisbon. Doubles an unofficial startup hub during Web Summit

> ## **Post-Event Actions**

Reviewer: Remove **

> ### **Reviewing and Organizing Notes**

Reviewer: Remove **

After the event, review and organize your notes. Highlight key takeaways and action items to implement what you’ve learned.

> ### **Following Up with Contacts**

Reviewer: Remove **

> Firstly, download a CSV of all your team’s connections from the Web Summit app, then follow up regularly by adding them to your CRM, like Hubspot or a similar tool."

Reviewer: ```suggestion
Firstly, download a CSV of all your team’s connections from the Web Summit app, then follow up regularly by adding them to your CRM, like Hubspot or a similar tool.
```

Send follow-up emails or LinkedIn messages to maintain the connections you made. Personalize each message using the notes you took to strengthen your new relationships.

E.g.

*Hey Mike,*

*I really enjoyed our chat about how refugees make great founders!!*

> ### **Implementing Learnings**

Reviewer: Remove **

Apply the insights and knowledge gained at Web Summit to your projects or business. Consider sharing your learnings internally with your team or externally through social media, a video or blog post. This not only helps solidify your learnings but also builds your personal brand.

> ## **Conclusion**

Reviewer: Remove **

### **Recap of Key Points**

Preparation is key to making the most of Web Summit. From planning your schedule to networking effectively, this guide has hopefully provided you with the tools to navigate the event successfully.

> ### **Final Encouragement**

Reviewer: Remove **

Web Summit is a unique opportunity to learn, grow, and connect. Approach it with enthusiasm and an open mind, and you’ll leave with valuable experiences and insights.

Have fun, but it’s not a party. You can do that when you get home after landing a big deal :)

**Congratulations!!**

You’ve read my whole guide, wow!!! So that gets you a ticket to our FusionAuth exclusive event on night one. Simply [fill in this form](https://lu.ma/m3k2n5hz) and I’ll personally send you an invite.

**See Us at Web Summit**

Please come and say hi to FusionAuth at our booth, or see me (Brendan) whilst I host the New Venture stage on Day 1 and the AI Summit on Day 2.

**Don’t know FusionAuth?**

FusionAuth adds login, registration, SSO, MFA, and a bazillion other features to your app in days - not months.

> Click [here](https://klever.kraftful.com/surveys-public/startups) to find out more and sign up to the waitlist for our exclusive program for early stage startups.

Reviewer: ```suggestion
[Click here](https://klever.kraftful.com/surveys-public/startups) to find out more and sign up to the waitlist for our exclusive program for early stage startups.
```

> **About the Author**

Reviewer: ```suggestion
```

> **Brendan Yell** is a Sydney, Australia based startup community builder and current co-host of the PITCH competition, New Venture Summit stage, and AI Stage at Web Summit. He also runs Startup Engagement at FusionAuth.

Reviewer: ```suggestion
```

Reviewer: we don't do about us text.

**Prepare for the Questions**

Prepare as much for the questions as you prepare for the pitch. A series of great succinct answers will help your chances a lot.

### Engaging with Startups and Companies

When engaging with startups, consider how their innovations could impact your industry. Look for potential collaborations or investment opportunities.

Don’t sell to booth owners, they have paid good money and they are selling *their* stuff. Sure, drop by and say hi and drop a card, but don’t abuse their time.

### Identifying Trends and Innovations

Pay attention to recurring themes and emerging trends. These insights can provide a competitive edge in your work or business.

At the 2022 Web Summit, there were a lot of mumbles about the future of AI. It wasn’t loud but it was there and recurring.

ChatGPT launched about 2 weeks later. The 2023 Summit was all about AI.

### Approaching the Media

We asked Mike Butcher, Techcrunch Editor-at-Large and Web Summit veteran for his tips on engaging with the media. Here they are:

> 1. Ping your journalist before the event with an email/contact to set-up a meeting or offer them a story exclusive so the story comes out whilst your are there.

Reviewer: ```suggestion
1. Ping your journalist before the event with an email/contact to set-up a meeting or offer them a story exclusive so the story comes out whilst you are there.
```

2. Scour the agenda to see where a journalist is moderating a panel or doing an interview. Try and grab them as they leave the stage.

3. "Goal hang" where the media enter the main venue from the VIP/Press area.

4. Repeat steps 2 & 3

> Make sure you follow Mike - @mikebutcher and check out [The Europas Awards](https://www.theeuropas.com/) an event he created which runs during Web Summit.

Reviewer: ```suggestion
Make sure you follow Mike - @mikebutcher and check out [The Europas Awards](https://www.theeuropas.com/), an event he created which runs during Web Summit.
```
